splunk_consts.py

```diff
@@ -39,7 +39,7 @@
     configuration and|or the action parameters."
 SPLUNK_EXCEPTION_ERROR_MESSAGE = "{msg}. Error Code: {error_code}. Error Message: {error_msg}"
 SPLUNK_JOB_FIELD_NOT_FOUND_MESSAGE = "{field} not found"
-
+SPLUNK_ERR_INVALID_VALUE_INTEGER = "Please provide value <= 120 seconds in the {param} parameter"
```

Comment:
```suggestion
SPLUNK_ERR_INVALID_VALUE_INTEGER = "Please provide a value <= 120 seconds in the {param} parameter"
```

minor: this constant name is bit misleading since it's not doing a general integer check. maybe something like `SPLUNK_ERR_INVALID_SLEEP_TIME`?